main/src/ui/occupant_menu/view.vala

```diff
@@ -102,10 +105,9 @@ public class View : Popover {
     }
 
     private void private_conversation_button_clicked() {
-        ListRow? list_row = list.list_box.get_selected_row() as ListRow;
-        if (list_row == null) return;
+        if (selected_jid == null) return;
 
-        Conversation conversation = stream_interactor.get_module(ConversationManager.IDENTITY).create_conversation(list_row.jid, list_row.conversation.account, Conversation.Type.GROUPCHAT_PM);
+        Conversation conversation = stream_interactor.get_module(ConversationManager.IDENTITY).create_conversation(selected_jid, conversation.account, Conversation.Type.GROUPCHAT_PM);
```

Comment:
Please also do those changes in the `kick_button_clicked` function.

Reply:
Fixed.